.ddev/.env

```diff
@@ -1 +1,4 @@
 FMS_API_SECRET="this is a test secret"
+FMS_API_AUTH_URL="ddev-forests-web/oauth/token"
+FMS_API_BASE_URL="ddev-forests-web/api"
```

Comment:
Isn't the change missing in the Dockerfiles/docker-compose? or we change this in portainer directly?

Reply:
I was originally going to change them directly in portainer but it's a better idea to set the, in docker-compose. I haven't done it yet because the NFA server is down and I can't test for the right values.